bot/taskcluster-hook.json

```diff
@@ -6,82 +6,101 @@
-        "routes": [],
-        "schedulerId": "-",
-        "scopes": [
-            "secrets:get:project/relman/code-coverage/runtime-CHANNEL",
-            "notify:email:*",
-            "docker-worker:cache:code-coverage-bot-CHANNEL",
-            "index:insert-task:project.releng.services.project.CHANNEL.code_coverage_bot.*"
-        ],
-        "tags": {},
-        "workerType": "releng-svc-memory"
+                    },
+                    "owner": "mcastelluccio@mozilla.com",
+                    "source": "https://github.com/mozilla/code-coverage"
+                },
+                "payload": {
+                    "artifacts": {
+                        "public/chunk_mapping.tar.xz": {
+                            "path": "/chunk_mapping.tar.xz",
+                            "type": "file"
+                        },
+                        "public/zero_coverage_report.json": {
+                            "path": "/zero_coverage_report.json",
+                            "type": "file"
+                        }
+                    },
+                    "cache": {
+                        "code-coverage-bot-CHANNEL": "/cache"
+                    },
+                    "capabilities": {},
+                    "command": [
+                        "code-coverage-bot",
+                        "--taskcluster-secret",
+                        "project/relman/code-coverage/runtime-CHANNEL",
+                        "--cache-root",
+                        "/cache"
+                    ],
+                    "env": {
+                        "$merge": [
+                            {
+                                "APP_CHANNEL": "CHANNEL"
+                            },
+                            {
+                                "$if": "firedBy == 'triggerHook'",
+                                "else": {},
+                                "then": {
+                                    "$eval": "payload"
+                                }
+                            }
+                        ]
+                    },
+                    "features": {
+                        "taskclusterProxy": true
+                    },
+                    "image": "mozilla/code-coverage:bot-REVISION",
+                    "maxRunTime": 14400
+                },
+                "priority": "normal",
+                "provisionerId": "aws-provisioner-v1",
+                "retries": 5,
+                "routes": [],
+                "schedulerId": "-",
+                "scopes": [
+                    "secrets:get:project/relman/code-coverage/runtime-CHANNEL",
+                    "notify:email:*",
+                    "docker-worker:cache:code-coverage-bot-CHANNEL",
+                    "index:insert-task:project.releng.services.project.CHANNEL.code_coverage_bot.*"
+                ],
+                "tags": {},
+                "workerType": "releng-svc-memory"
+            }
+        ]
     },
+    "schedule": [
```

Comment:
Nit: why did you move the schedule?

Reply:
I did not, it's just that the alphabetical order puts it after "$merge" now